README.md

```diff
@@ -1,73 +1,189 @@
+<a name="readme-top"></a>
 ![](https://img.shields.io/badge/Microverse-blueviolet)
```

Comment:
[REQUIRED] Kindly fix or remove this logo because it is not displaying properly.